docs/README.md

```diff
@@ -0,0 +1,6 @@
+# Documentation
+
+- [Modelfile](./modelfile.md)
+- [How we store Models](./modelstorage.md)
```

Comment:
This links to a non-existent page

Reply:
oh, yeah. it will. but will remove for now